lib/chef/knife/rackspace_server_create.rb

```diff
@@ -460,11 +482,35 @@ def run
         msg_pair("Environment", config[:environment] || '_default')
         msg_pair("Run List", config[:run_list].join(', '))
       end
-      
+
+      def setup_rackconnect_network!(server)
+        # I know. I know. It beats another Identity call
+        auth_token = connection.instance_variable_get("@auth_token")
+        tenant_id  = connection.instance_variable_get("@uri").path[4..-1]
```

Comment:
You can get this without being so invasive: `tenant_id = c.endpoint_uri.path.split("/").last`